packages/gatsby-theme-docs/color-presets/index.js

```diff
@@ -26,4 +26,19 @@ module.exports = {
     relativePath: 'platform-developer-docs',
     value: require('./platform-developer-docs'),
   },
+  optionOne: {
+    key: 'optionOne',
+    relativePath: 'option-1',
+    value: require('./option-1'),
+  },
+  optionTwo: {
+    key: 'optionTwo',
+    relativePath: 'option-2',
+    value: require('./option-2'),
+  },
+  optionThree: {
+    key: 'optionThree',
+    relativePath: 'option-3',
```

Comment:
I don't know where it comes from but the heading text color is awful (at least in the VRT shots).  Option 3 is the only one that does not have `#003037` as the primary color, I'm sure this was not intentional.  Can you change it to the same color as the other schemes in option 3 please so that the theme only has the banner and its background as a scheme?

Reply:
Sure. Do we still want to proceed with the 3rd option for the terms website?